### ISSUE TITLE
organize data on R x S study and prepare to publish

### ISSUE THREAD

**Issue by lizzieinvancouver** [lizzieinvancouver/asevdiversitydup#5]
We should publish the data we scraped! Usually we publish on [KNB](https://knb.ecoinformatics.org/). We could there also publish information on how we scraped etc. There's NO RUSH to do this (it would be best to have it done before publication... and we have not submitted for review yet, so it's many months off). But can be good to do soon while all the methods are fresh in our minds. Let me know if this is something you could help with.

**Comment by christophe-rd** [comment]
@lizzieinvancouver I could definitely help with this. Do you have any precisions on the way I should write the information on how we scrapped? Should I just write a normal methodology text?

**Comment by lizzieinvancouver** [comment]
@christophe-rd Yes! Usually there is just a methods box we can paste text into in KNB. [Here's an example](https://knb.ecoinformatics.org/view/doi:10.5063/F1CZ35KB) and if I can help more, let me know.

**Comment by lizzieinvancouver** [comment]
@christophe-rd Thanks for your ms comments! I am going through them now and they're great. In regards to this one:

> Should we also mention that we extracted papers that were used in review papers?

Let's put this in the methods section of the data publication. We're already a little long on the caption and have not added the interactive html (let me know when that's ready).

**Comment by christophe-rd** [comment]
> Let's put this in the methods section of the data publication. We're already a little long on the caption and have not added the interactive html (let me know when that's ready)

Perfect! I already started working on the method, and will keep working on it this week! Could you please let me know if you need it soon?

**Comment by lizzieinvancouver** [comment]
@christophe-rd No rush on the methods at all! We just need to data and methods ready to go for publication ... which is WAY far away seeing as we have not submitted. But it's good to write these things before we forget. Do you think you could have the methods done by 15 October perhaps?

**Comment by lizzieinvancouver** [comment]
@christophe-rd Also I will work out where to finalize the interactive link with Tolu.

**Comment by christophe-rd** [comment]
> @christophe-rd No rush on the methods at all! We just need to data and methods ready to go for publication ... which is WAY far away seeing as we have not submitted. But it's good to write these things before we forget. Do you think you could have the methods done by 15 October perhaps?

Yes that's perfect!

**Comment by christophe-rd** [comment]
> @christophe-rd Also I will work out where to finalize the interactive link with Tolu.

Perfect! She emailed me yesterday. I will answer her and CC you.

**Comment by christophe-rd** [comment]
Hi @lizzieinvancouver ! I wrote a first draft for methodology of asevdiversity. I decided to give latex a try. It was a first for me. [You will find the text here](https://github.com/lizzieinvancouver/asevdiversitydup/blob/c5a9b40cde0c8bd66e55e6e96dc1aceca3f0e049/methodology_text.tex).
The first section is the description of the methods (please feel free to remind me of any sections I would've forgotten). 
The second section is _Data Table, Image, and Other Data Details_ which I found in the example you provided. 

I also cleaned and deleted any unnecessary lines from the choropleth map code. [It's here](https://github.com/lizzieinvancouver/asevdiversitydup/blob/c5a9b40cde0c8bd66e55e6e96dc1aceca3f0e049/rootstockxscion/analyses/choropleth_script.R)

**Comment by lizzieinvancouver** [comment]
@christophe-rd Sorry for dropping the ball on this. Can you post a PDF for me to read? I usually look at a PDF while editing tex files.

**Comment by christophe-rd** [comment]
Hi @lizzieinvancouver! No problem. [You can find it here.](https://github.com/lizzieinvancouver/asevdiversitydup/blob/546d0b9c48e6e6de483a0f9efa3c245a9e3bf149/methodology_text.pdf)

**Comment by lizzieinvancouver** [comment]
@christophe-rd Thanks! I finally did this. I made some text edits for clarity, but looks very good. Once you fix the edits (add url and some column names) I think we're all set. You'll probably want to paste in the methods in a methods box on KNB and also upload the pdf since it's nice to read -- just note in both places that the information is duplicated so people don't review each and try to figure it out. I think we can wait for reviews to post (especially the code as the reviewers may ask for edits to the figure).

**Comment by christophe-rd** [comment]
@lizzieinvancouver Here's the KNB box (I believe this is what I created!). As this was also a first for me, I did my best, but not too sure if that's OK. Could you please let me know anything I should change?
Here's the urn: urn:uuid:a72aa0e5-2ad7-4658-a0a0-23937acef477
If the link above doesn't work: https://knb.ecoinformatics.org/view/urn%3Auuid%3Aa72aa0e5-2ad7-4658-a0a0-23937acef477
